sql/core/src/test/resources/sql-tests/results/typeCoercion/native/concat.sql.out

```diff
@@ -1,5 +1,5 @@
 -- Automatically generated by SQLQueryTestSuite
--- Number of queries: 11
+-- Number of queries: 14
```

Comment:
Not sure why this changed...

Reply:
Someone probably only updated the contents of the file and not this summary.